src/main/java/com/conveyal/datatools/manager/auth/Auth0Users.java

```diff
@@ -48,20 +49,20 @@
     /**
      * Overload of user search query URL to restrict search to current users only.
      */
-    private static URI getUrl(String searchQuery, int page, int perPage, boolean includeTotals) {
-        return getUrl(searchQuery, page, perPage, includeTotals, true);
+    private static URI getSearchUrl(String searchQuery, int page, int perPage, boolean includeTotals) {
+        return getSearchUrl(searchQuery, page, perPage, includeTotals, true);
     }
 
     /**
      * Constructs a user search query URL.
      * @param searchQuery   search query to perform (null value implies default query)
      * @param page          which page of users to return
-     * @param perPage       number of users to return per page
+     * @param perPage       number of users to return per page (limited to 1000)
```

Comment:
Add comment on where does the limitation come from?

Reply:
Fixed in 51e8026486a3ec7adeca35444727da603b853ec6